addons/website_hr_recruitment/controllers/main.py

```diff
@@ -297,11 +297,12 @@ def _build_search_domain(self, field, value):
     def _get_ongoing_application_response(self, ongoing_application, job_id, value):
         message = _("An application already exists for %s. Duplicates might be rejected.", value)
         if ongoing_application.user_id:
-            message += " " + _("In case of issue, contact %s", ongoing_application.user_id.name)
+            contact = ongoing_application.user_id.name
             if ongoing_application.user_id.email:
-                message += ", %s" % ongoing_application.user_id.email
+                contact += ", %s" % ongoing_application.user_id.email
             if ongoing_application.user_id.phone:
-                message += ", %s" % ongoing_application.user_id.phone
+                contact += ", %s" % ongoing_application.user_id.phone
```

Comment:
The appropriate delimiter changes depending on the language. Perhaps you could create a list of all the contact details, and then merge them using `format_list`